### PR TITLE
Update to 3.2.0

### PR DESCRIPTION
**Pull request by nicolas377**
Please fill in this template.

- [x] Use a meaningful title for the pull request.
- [x] Test the change in your own code. (Compile and run.)
- [x] Run `npm build`.
- [x] Run `npm test`.
- [x] Passed internal testing.

Changelog:

- Optimized AP++ file size by loading nav data from chrome extension. #42
- Added support for GeoFS betas.
- Prevented scripts from injecting when there's no permission to do so.
- Made sure scripts were injected in the right order. #56
- Fixed issue where VNAV broke below 1500 feet. https://github.com/GeoFS-Extensions/fmc-requirejs/issues/11
- Made updates quieter.

### CODE REVIEW

**Comment by nicolas377** [comment]
The rebase screwed with the css, so that made the popup look, shall we say interesting.
Also, some images needed updating.
Deploying a fix right now.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+## Contributing
+
+First off, thanks for considering contributing to Autoland! It's people like you that make this project a great tool.
```
Comment (Guy-Adler) [inline CONTRIBUTING.md:3]: Needs to be changed from Autoland to GeoFS Plugins

**Review by Guy-Adler — COMMENTED** [review]
In some places Autoland needs 
 to be replaced with GeoFS Plugins.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nicolas377) [reply CONTRIBUTING.md:3]:
```suggestion
First off, thanks for considering contributing to this project! It's people like you that make this project a great tool.
```
How does that look?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+### Where do I go from here?
+
+If you've noticed a bug or have a feature request, [make one]! It's
+generally best if you get confirmation of your bug or approval for your feature
+request this way before starting to code.
+
+### Fork & create a branch
+
+If this is something you think you can fix, then [fork Autoland] and create
+a branch with a descriptive name.
```
Comment (nicolas377) [inline CONTRIBUTING.md:14]:
```suggestion
If this is something you think you can fix, then [fork this project] and create
a branch with a descriptive name.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+- The last digit (e.g. 3.2.3.**14**) is for CI releases, and you should reset that number to 0.
+- The second to last digit (e.g. 3.2.**3**.14) is for patches. If you're fixing a bug, increment this number. (e.g. 3.2.4.0)
+- The second digit (e.g. 3.**2**.3.14) is for new features. If you're adding a new option, or adding a new functionality to
+  a script, increment this digit and reset the digits right of it to 0. (3.3.0.0)
+- The first digit (e.g. **3**.2.3.14) is for very major releases. If your change doesn't fall into the other catagories,
+  increment this digit and reset the digits right of it to 0. You should also open a discussion for this. (e.g. 4.0.0.0)
+
+### Make a Pull Request
+
+At this point, you should switch back to your main branch and make sure it's up to date with Autoland's main branch:
```
Comment (nicolas377) [inline CONTRIBUTING.md:56]:
```suggestion
At this point, you should switch back to your main branch and make sure it's up to date with our main branch:
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+```sh
+git remote add upstream git@github.com:geofs-autoland/autoland.git
```
Comment (nicolas377) [inline CONTRIBUTING.md:59]:
```suggestion
git remote add upstream git@github.com:geofs-extensions/autoland.git
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+- It has no requested changes.
+- It is up to date with current main.
+
+Any maintainer is allowed to merge a PR if all of these conditions are met.
+
+Thanks to the [Active Admin Contributing guide] for inspiring this Contributing guide!
+
+[active admin contributing guide]: https://github.com/activeadmin/activeadmin/blob/HEAD/CONTRIBUTING.md
+[make one]: https://github.com/GeoFS-Autoland/autoland/issues/new/choose
+[fork autoland]: https://help.github.com/articles/fork-a-repo
```
Comment (nicolas377) [inline CONTRIBUTING.md:110]:
```suggestion
[make one]: https://github.com/GeoFS-Extensions/autoland/issues/new/choose
[fork this project]: https://help.github.com/articles/fork-a-repo
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<h1 align="center">GeoFS Plugins</h1>
+
+<p align="center">
+  <a href="https://github.com/GeoFS-Autoland/autoland/actions/workflows/tests.yml">
+    <img alt="tests" src="https://img.shields.io/github/workflow/status/GeoFS-Autoland/autoland/Tests?label=Tests&style=flat-square"></a>
```
Comment (nicolas377) [inline README.md:5]:
```suggestion
  <a href="https://github.com/GeoFS-Extensions/autoland/actions/workflows/tests.yml">
    <img alt="tests" src="https://img.shields.io/github/workflow/status/GeoFS-Extensions/autoland/Tests?label=Tests&style=flat-square"></a>
```

**Review by nicolas377 — COMMENTED** [review]
Here's some of the things I need to change when I get home.

**Review by Guy-Adler — APPROVED** [review]

**Review by Guy-Adler — APPROVED** [review]

**Pull request merged**